tests/pages/system/settings/UISettings-cy.js

```diff
@@ -0,0 +1,319 @@
+describe("UI Settings", function() {
+  if (!Cypress.env("FULL_INTEGRATION_TEST")) {
```

Comment:
So this env is present only for ee tests?

Reply:
You can run integration tests against an actual cluster by setting that env variable to true. But the tests I wrote would all fail in that scenario b/c I'm waiting on mocked API calls and updating how the API responds in tests in some cases.